Libplanet.Net/Transports/NetMQTransport.cs

```diff
@@ -871,33 +906,41 @@ private async Task ProcessRequest(MessageRequest req, CancellationToken cancella
                 }
 
                 tcs.TrySetResult(result);
+                const string logMsg =
+                    "Request {Message} {RequestId} with timeout {TimeoutMs:F0}ms " +
+                    "processed in {DurationMs:F0}ms.";
+                _logger
+                    .ForContext("Tag", "Metric")
+                    .Debug(
+                    logMsg,
+                    req.Message,
+                    req.Id,
+                    req.Timeout is TimeSpan t ? t.TotalMilliseconds : 0.0,
+                    (DateTimeOffset.UtcNow - startedTime).TotalMilliseconds);
             }
-            catch (MessageSendFailedException msfe)
-            {
-                tcs.TrySetException(msfe);
-            }
-            catch (DifferentAppProtocolVersionException dapve)
-            {
-                tcs.TrySetException(dapve);
-            }
-            catch (InvalidMessageTimestampException imte)
-            {
-                tcs.TrySetException(imte);
-            }
-            catch (InvalidMessageSignatureException imse)
+            catch (Exception e) when (
+                e is DifferentAppProtocolVersionException ||
+                e is InvalidMessageTimestampException ||
+                e is TimeoutException)
```

Comment:
This should also catch `MessageSendFailedException` and `InvalidMessageSignatureException`.

Reply:
```suggestion
            catch (Exception e) when (
                e is DifferentAppProtocolVersionException ||
                e is InvalidMessageTimestampException ||
                e is InvalidMessageSignatureException ||
                e is MessageSendFailedException ||
                e is TimeoutException)
```

is it right?